src/schemas/complaint-tool/schema.js

```diff
@@ -68,6 +72,86 @@ const fullName = _.set('properties.suffix', {
   ]
 }, definitions.fullName);
 
+const USA = countries.find(country => country.value === 'USA');
+const nonUSACountries = countries.filter(country => country.value !== 'USA');
+
+const domesticSchoolAddress = {
+  type: 'object',
+  required: ['street', 'city', 'state', 'country', 'postalCode'],
+  properties: {
+    street: {
+      type: 'string',
+      minLength: 1,
+      maxLength: 126 // street + street2 length must be < 255
+    },
+    street2: {
+      type: 'string',
+      minLength: 1,
+      maxLength: 126 // street + street2 length must be < 255
+    },
+    city: {
+      type: 'string',
+      minLength: 1,
+      maxLength: 255
+    },
+    state: {
+      type: 'string',
+      'enum': allStates.map(state => state.abbreviation), // backend accepts abbreviated state names
+      enumNames: allStates.map(state => state.full)
+    },
+    postalCode: {  // TYPE: text (255)
+      type: 'string',
+      pattern: '^\\d{5}$' // common definition pattern (meets submission requirements)
+    },
+    country: {
+      type: 'string',
+      'enum': [USA].map(country => country.value),
+      enumNames: [USA].map(country => country.label),
+      default: USA.value
+    }
+  }
+};
+
+// TRANSLATE: all international address fields into street, street2, and country
+const internationalSchoolAddress = {
```

Comment:
These maxLengths add up to 249, with 5 left over for spaces and commas to be inserted. Does this breakdown of limits between seem reasonable?

We previously discussed using `street`, `street2`, `city`, and `country` for international addresses (and only removing `state` and `postal code` from international addresses), but putting `state` and `postal code` onto `street2`, but leaving `city` in the `city` field, seems like it would be unclear to read, and these limits are set up with the idea that we will also move `city` into `street2` in mind.

Reply:
Seems reasonable